src/main/java/com/app/bookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.app.bookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
```

Comment:
If we name this class as "MapperConfiguration" we can use @MapperConfig annotation